components/time.rst

```diff
@@ -10,7 +10,184 @@ Time
+  Range is from 1 (January) to 12 (December).
+- **days_of_week** (*Optional*, string): Specify for which days of the week to trigger.
+  Defaults to ``*`` (all days). The names SUN to SAT are automatically substituted.
+  Range is from 1 (Sunday) to 7 (Saturday).
+- **cron** (*Optional*, string): Alternatively, you can specify a whole cron expression like
+  ``* /5 * * * *``. Please note years and some special characters like ``L``, ``#`` are currently not supported.
+
+- See :ref:`Automation <automation>`.
+
+In the ``seconds:``, ``minutes:``, ... fields you can use the following operators:
+
+- .. code-block:: yaml
+
+      seconds: 0
+
+  An integer like ``0`` or ``30`` will make the automation only trigger if the current
+  second is **exactly** 0 or 30, respectively.
+- .. code-block:: yaml
+
+     seconds: 0,30,45
+
+  You can combine multiple expressions with the ``,`` operator. This operator makes it so that
+  if either one of the expressions separated by a comma holds true, the automation will trigger.
+  For example ``0,30,45`` will trigger if the current second is either ``0`` or ``30`` or ``45``.
+- .. code-block:: yaml
+
+      days_of_week: 2-6
+      # same as
+      days_of_week: MON-FRI
+      # same as
+      days_of_week: 2,3,4,5,6
+      # same as
+      days_of_week: MON,TUE,WED,THU,FRI
+
+  The ``-`` (hyphen) operator can be used to create a range of values and is shorthand for listing all
+  values with the ``,`` operator.
+- .. code-block:: yaml
+
+      # every 5 minutes
+      seconds: 0
+      minutes: /5
+
+      # every timestamp where the minute is 5,15,25,...
+      seconds: 0
+      minutes: 5/10
+
+  The ``/`` operator can be used to create a step value. For example ``/5`` for ``minutes:`` makes an
+  automation trigger only when the minute of the hour is 0, or 5, 10, 15, ... The value in front of the
+  ``/`` specifies the offset with which the step is applied.
+
+- .. code-block:: yaml
+
+      # Every minute
+      seconds: 0
+      minutes: '*'
+
+  Lastly, the ``*`` operator matches every number. In the example above, ``*`` could for example be substituted
+  with  ``0-59``.
+
+.. warning::
+
+    Please note the following automation would trigger for each second in the minutes 0,5,10,15 and not
+    once per 5 minutes as the seconds variable is not set:
+
+    .. code-block:: yaml
+
+        time:
+          - platform: sntp
+            # ...
+            on_time:
+              - minutes: /5
+                then:
+                  - switch.toggle: my_switch
+
+.. _time-on_time_sync:
+
+``on_time_sync`` Trigger
+************************
+
+This automation can be used to run synchronize several real time clocks.
+See the :ref:`DS1307 configuration example <ds1307-config_example>` for a scenario
+where a network time synchronization from a home assistant server trigger a write
+to an external hardware real time clock chip.
```

Comment:
Also this explanation can be improved, the example is good and makes sense, but before that I get confused. 

```suggestion
This automation is triggered after a time source successfully retrieves the current time.
See the :ref:`DS1307 configuration example <ds1307-config_example>` for a scenario
where a network time synchronization from a home assistant server trigger a write
to an external hardware real time clock chip.
```